crates/turbopack-node/src/transforms/postcss.rs

```diff
@@ -186,12 +186,11 @@ impl Asset for PostCssTransformedAsset {
         )
         .await?;
         let JavaScriptValue::Value(val) = &*config_value else {
-            bail!("Expected a value from PostCSS transform");
```

Comment:
we don't want this error?

Reply:
<img width="1280" alt="Screen Shot 2022-12-19 at 4 06 07 PM" src="https://user-images.githubusercontent.com/112982/208523930-c0ac08b2-1989-4eca-b82f-db5e2674ac3e.png">